File: lib/assets/patches/DidFinishLaunching.m
<% if use_conditional_test_key? %>
#ifdef DEBUG
    [Branch setUseTestBranchKey:YES];
#endif // DEBUG
<% end %>
    [[Branch getInstance] initSessionWithLaunchOptions:launchOptions
        andRegisterDeepLinkHandlerUsingBranchUniversalObject:^(BranchUniversalObject *universalObject, BranchLinkProperties *linkProperties, NSError *error){
        // TODO: Route Branch links
    }];

File: lib/assets/patches/DidFinishLaunching.swift
<% unless config.keys.count <= 1 || has_multiple_info_plists? >
        #if DEBUG
            Branch.setUseTestBranchKey(true)
        #endif

<% end %>
        Branch.getInstance().initSession(launchOptions: launchOptions) {
            universalObject, linkProperties, error in

            // TODO: Route Branch links
        }
